From 05f280476dbac852ef84d74cbefe297eb4d517bf Mon Sep 17 00:00:00 2001
From: Violet Purcell <vimproved@inventati.org>
Date: Tue, 5 Sep 2023 08:04:43 -0400
Subject: [PATCH] Fix build with clang on linux

Clang does not support the -fno-gnu-unique argument. Previously this was
not added if BSD was true (presumably to work around FreeBSD support)
but there is a CLANG variable serving that purpose. Use that instead.

Signed-off-by: Violet Purcell <vimproved@inventati.org>
--- a/source/Makefile.mk
+++ b/source/Makefile.mk
@@ -85,7 +85,7 @@ CXXFLAGS   += -fvisibility-inlines-hidden
 endif
 
 ifneq ($(MACOS_OR_WASM_OR_WINDOWS),true)
-ifneq ($(BSD),true)
+ifneq ($(CLANG),true)
 BASE_FLAGS += -fno-gnu-unique
 endif
 endif
